From 02a9f17fe8af026a370460e9160d36a8fa3a715d Mon Sep 17 00:00:00 2001
From: Lijun Pan <Lijun.Pan@freescale.com>
Date: Mon, 8 Feb 2016 17:40:15 -0600
Subject: [PATCH 11/12] staging: fsl-mc: fsl-mc bus's sysfs rescan
 documentation
Patch-mainline: Queued in subsystem maintainer repository
Git-repo: https://github.com/qoriq-open-source/linux.git nxp/linux-v4.5
Git-commit: 02a9f17fe8af026a370460e9160d36a8fa3a715d
References: fate#319900

Add the sysfs documentation for the rescan attribute of
fsl-mc bus.

Signed-off-by: Lijun Pan <Lijun.Pan@freescale.com>
Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 Documentation/ABI/testing/sysfs-bus-fsl-mc | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 Documentation/ABI/testing/sysfs-bus-fsl-mc

diff --git a/Documentation/ABI/testing/sysfs-bus-fsl-mc b/Documentation/ABI/testing/sysfs-bus-fsl-mc
new file mode 100644
index 0000000..3fa747c
--- /dev/null
+++ b/Documentation/ABI/testing/sysfs-bus-fsl-mc
@@ -0,0 +1,11 @@
+What:		/sys/bus/fsl-mc/rescan
+Date:		October 2015
+Contact:	Lijun Pan <Lijun.Pan@freescale.com>
+Description:
+		Writing a non-zero value to this attribute will
+		force a rescan of fsl-mc bus in the system, and
+		add/remove previously added/removed objects in
+		Management Complex firmwarer, i.e., synchronize
+		the objects under fsl-mc bus and Management Complex
+		firmware. For example:
+		# echo 1 > /sys/bus/fsl-mc/rescan
-- 
2.6.6

